docs/pages/reference/access-controls/access-monitoring-rules.mdx

```diff
@@ -0,0 +1,271 @@
+---
+title: Access Monitoring Rule Reference
+description: An explanation and overview of Access Monitoring Rules.
+---
+
+Access Monitoring Rules allow administrators to monitor Access Requests and apply
+notification routing rules or automatic review rules based on specific conditions.
+
+An Access Monitoring Rule can configure both a notification routing rule and an
+automatic review rule, or it can contain only one.
+
+## YAML specification
+
```

Comment:
I would add a brief introduction to the section here to save the reader a few cycles:

```suggestion

An Access Monitoring Rule is a dynamic Teleport resource with a structure similar to the following:

```